patch/llpcPatchPeepholeOpt.cpp

```diff
@@ -655,7 +655,9 @@ void PatchPeepholeOpt::visitPHINode(
         }
     }
 
-    if (pPrevIncomingInst != nullptr)
+
+    // Do not clone allocas -- we don't want to potentially introduce them in the middle of the function.
+    if (pPrevIncomingInst != nullptr && !isa<AllocaInst>(pPrevIncomingInst))
```

Comment:
if (pPrevIncomingInst != nullptr && !isa<AllocaInst>(pPrevIncomingInst))
->
if ((pPrevIncomingInst != nullptr) && (isa<AllocaInst>(pPrevIncomingInst) == false))

(1) Each condition with relational statement ought to have a "( )"
(2) We don't use negative expression